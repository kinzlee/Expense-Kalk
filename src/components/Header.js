import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expense Kalk</h1>
        <NavLink to="/" className="is-active" exact={true}>Dashboard</NavLink>
        <NavLink to="/expense" className="is-active">Expense page</NavLink>
        <NavLink to="/edit" className="is-active">edit expense page</NavLink>
        <NavLink to="/help" className="is-active"> help</NavLink>
    </header>
);

export default Header;