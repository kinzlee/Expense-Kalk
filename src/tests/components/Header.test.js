import React from 'react';
import {shallow} from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header';

test('should render header appropriately', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();


    // expect(wrapper.find('h1').text()).toBe('Expense Kalk');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header/>);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});