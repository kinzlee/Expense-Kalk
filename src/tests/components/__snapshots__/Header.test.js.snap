// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header appropriately 1`] = `
<header>
  <h1>
    Expense Kalk
  </h1>
  <NavLink
    className="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    className="is-active"
    to="/expense"
  >
    Expense page
  </NavLink>
  <NavLink
    className="is-active"
    to="/edit"
  >
    edit expense page
  </NavLink>
  <NavLink
    className="is-active"
    to="/help"
  >
     help
  </NavLink>
</header>
`;
